### DISCUSSION TITLE
Running ASReview on Android phone

### DISCUSSION THREAD

**Discussion by NIOZ-QingZ** [asreview/asreview#1347] [Q&A]
From the website video, it seems possible to run ASReview on mobile phon ([https://www.youtube.com/watch?v=LLLVYNqauhM](https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DLLLVYNqauhM&data=05%7C01%7CQ.Zhan%40nioo.knaw.nl%7C855dd057b93a45120c1908db25f56051%7Cc7a0410b695b4223af74d14c2e7652e8%7C0%7C0%7C638145508052100749%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=AUuI1Xsi2olBqQ0heGuDmgXZClR3nVHv%2BLYtpAAW1wY%3D&reserved=0))? Anyone any idea how to do it?

**Comment by jteijema** [comment]
Hi @NIOO-QingZ,

To use ASReview on your mobile device, you can follow the method I use:

- Set up ASReview on your desktop (or other service device capable of running ASReview)
- Find out the local IP address of ASReview
- Run ASReview Lab, setting ` --ip IP`
- Connect your phone to ASReview via the browser

This works for a local network only. There isn't a stand-alone version for mobile devices.

**Reply by NIOZ-QingZ** [reply]
THank you for your answers! But I had difficulty obtaining the local IP. In the ASReview browser website setting, I don't see ip information. Could you also illustrate how to connect a phone to ASReview via the browser? THANKS!

**Reply by NIOZ-QingZ** [reply]
Nevermind my previous quesiton. I achieve it by chrome remote desktop control.
